### ISSUE TITLE
`defaultValue` is greyed out

### ISSUE THREAD

**Issue by syky27** [n4kz/react-native-material-textfield#191]
Hi, 

Thank you for bringing this back to life, great job!

Anyways, when I use `defaultValue` to get updates as the var changes, the Textfield value is greyedout.

This is what it looks like
<img width="362" alt="Screenshot 2019-10-23 01 48 54" src="https://user-images.githubusercontent.com/2771802/67344135-8a12b900-f537-11e9-9ac2-66b45636e6a1.png">
This is what I think it should look like

<img width="363" alt="Screenshot 2019-10-23 01 49 01" src="https://user-images.githubusercontent.com/2771802/67344126-85e69b80-f537-11e9-894a-48b1a2a22ef7.png">

Thank you!

**Comment by shamilovtim** [comment]
Same issue here. There's no reason to present a value as a placeholder, and no way to alter the color of it without changing the whole color of the input.

**Comment by greatwitenorth** [comment]
@syky27 Did you ever figure out a solution to this?

**Comment by shamilovtim** [comment]
just use my fork

**Comment by Fallup** [comment]
Is this going to be merged?

**Comment by shamilovtim** [comment]
The repo doesn't have commits since oct. Feel free to use my fork, it has this commit.

**Comment by Fallup** [comment]
> The repo doesn't have commits since oct. Feel free to use my fork, it has this commit.

Yup, I saw the repo status, but wanted to ask anyway. Thanks, but I'm already using the fix with help of [patch-package](https://www.npmjs.com/package/patch-package) which I prefer more than foreign repos.

**Comment by syky27** [comment]
@greatwitenorth sorry, not without hotfixing this package. And decided I am not touching react native ever again. 🤢

**Comment by rahmath30** [comment]
@greatwitenorth just give the style={{color:"#000"}} that works for me
